tests/test_contrib_to_marc21_utils.py

```diff
@@ -55,3 +56,21 @@ def test_entry_points():
     output = dump(data,
                   xslt_filename='{0}/demo_marc21_to_dc.xslt'.format(path))
     assert output.decode('utf-8') == expect
+
+
+def test_output_type_from_dumps_etree():
```

Comment:
Actually this test is specific to test the type because I need to be sure is something that I can append to another `lxml` object. See https://github.com/jirikuncar/invenio-oaiserver/pull/2/files#diff-11013101f610d5f991e58c323609775dR273
We can add other tests to check if the result is a correct `xml`.